src/server/components/lists/controllers/index.ts

```diff
@@ -75,8 +89,23 @@ export async function listsPostController(
     }
   }
 
+  if (newLanguage) {
+    languagesProvided = setLanguagesProvided(newLanguage, languagesProvided as string);
+    params.languagesProvided = languagesProvided;
+  }
+  // @ts-ignore
+  if (params?.continueButton) {
+    // @ts-ignore
+    delete params.continueButton;
+  }
```

Comment:
Out of curiosity why is there a `@ts-ignore` here?

Reply:
continueButton wasn't added to ListsRequestParams so it gave a linting error.  I've added it now and no longer need the @ts-ignore.  Good catch.